### PR TITLE
Add things to send commands to server containers

### PR DESCRIPTION
**Pull request by KiyoNetcat**
Couldn't test locally as I can't create servers with the setup i have
But basically this should let you send any mc command to a container on the server's page

### CODE REVIEW

File: src/routes/[serverID]/+page.svelte
Comment (TenBitMelon) [inline src/routes/[serverID]/+page.svelte:1]: From what I can tell, the logs don't get the result of the command, just the command result.
Plus no code is returned from the command endpoint, just `{ success: boolean }`.

File: src/routes/api/[serverID]/command/+server.ts
Comment (TenBitMelon) [inline src/routes/api/[serverID]/command/+server.ts:1]: It would be better if the endpoint also returned the result of the command.

**Comment by TenBitMelon** [comment]
Other than that minor output tweak, the rest looked good.

**Comment by KiyoNetcat** [comment]
don't think github saved your review, not seeing it here unfortunately

**Comment by TenBitMelon** [comment]
There we go. I have learned how GitHub reviews work. So, other than the stuff in the review, it looks good.

**Comment by KiyoNetcat** [comment]
alright, changed it so that the api directly returns the output from the command and the page sets the message to the command output

**Pull request merged**